alembic/versions/2025_03_06_3a0250e5526d_add_nallo_to_limitations_table.py

```diff
@@ -0,0 +1,54 @@
+"""add-nallo-to-limitations-table
+
+Revision ID: 3a0250e5526d
+Revises: 6f368f57df4a
+Create Date: 2025-03-06 11:07:29.158959
+
+"""
+
+from sqlalchemy.dialects import mysql
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "3a0250e5526d"
+down_revision = "6f368f57df4a"
+branch_labels = None
+depends_on = None
+
+
+new_workflow = "nallo"
+
+old_options = (
+    "balsamic",
+    "balsamic-pon",
+    "balsamic-qc",
```

Comment:
Balsamic qc has been deprecated
```suggestion
```

Reply:
Ignoring this comment per our huddle. balsamic-qc will have to be removed with an alembic revision.